Sources/SwiftSyntaxBuilderGeneration/main.swift

```diff
@@ -14,172 +14,29 @@ import Foundation
-          })
-        } else {
-          let signature = FunctionSignature(input: ParameterClause(parameterList: FunctionParameter(attributes: nil, firstName: .wildcard, secondName: .identifier("text"), colon: .colon, type: "String"), rightParen: .rightParen.withTrailingTrivia(.spaces(1))), output: "TokenSyntax")
-
-          FunctionDecl(identifier: .identifier("`\(token.name.withFirstCharacterLowercased)`"),
-                       signature: signature,
-                       modifiersBuilder: {
-            if let text = token.text {
-              TokenSyntax.static.withLeadingTrivia(.newlines(1) + .docLineComment("/// The `\(text)` token"))
-            } else {
-              TokenSyntax.static
-            }
-          },
-                       bodyBuilder: {
-            FunctionCallExpr(MemberAccessExpr(base: "SyntaxFactory", name: "make\(token.name)"), argumentListBuilder: {
-              TupleExprElement(expression: IdentifierExpr("text"))
-            })
-
-            if token.requiresLeadingSpace {
-              createWithLeadingTriviaCall()
-            }
-
-            if token.requiresTrailingSpace {
-              createWithTrailingTriviaCall()
-            }
-          })
-        }
-      }
-      VariableDecl(letOrVarKeyword: .var,
-                   modifiersBuilder: { TokenSyntax.static.withLeadingTrivia(.newlines(1) + .docLineComment("/// The `eof` token") + .newlines(1)) },
-                   bindingsBuilder: {
-        // We need to use `CodeBlock` here to ensure there is braces around.
-        let body = CodeBlock {
-          FunctionCallExpr(MemberAccessExpr(base: "SyntaxFactory", name: "makeToken"),
-                           argumentListBuilder: {
-            TupleExprElement(expression: MemberAccessExpr(name: "eof"), trailingComma: .comma)
-            TupleExprElement(label: TokenSyntax.identifier("presence"), colon: .colon, expression: MemberAccessExpr(name: "present"))
-          })
-        }
-
-        createTokenSyntaxPatternBinding("eof", accessor: body)
-      })
-      VariableDecl(letOrVarKeyword: .var,
-                   modifiersBuilder: { TokenSyntax.static.withLeadingTrivia(.newlines(1) + .docLineComment("/// The `open` contextual token") + .newlines(1)) },
-                   bindingsBuilder: {
-        // We need to use `CodeBlock` here to ensure there is braces around.
-        let body = CodeBlock {
-          FunctionCallExpr(MemberAccessExpr(base: "SyntaxFactory", name: "makeContextualKeyword"),
-                           argumentListBuilder: {
-            TupleExprElement(expression: StringLiteralExpr("open"))
-          })
-
-          createWithTrailingTriviaCall()
-        }
-
-        createTokenSyntaxPatternBinding("open", accessor: body)
-      })
-    })
-}
-
-let tokenSyntax = tokensFile.buildSyntax(format: format)
+/// SwiftSyntaxBuilder sources to be generated
+let sourceTemplates = [
+  (tokensFile, "Tokens.swift"),
+]
```

Comment:
Could we load these files at runtime?

Reply:
You mean in a similar way to how test discovery works? I am not sure whether that would be possible on Linux

Reply:
Nope. 
Just use `FileManager` to find all files in the `template` folder. :)

Reply:
Ah... it's me. That's not possible as it's compiled 🤦‍♂️🤦‍♂️
Sorry for the confusion